.bazelrc

```diff
@@ -111,6 +111,10 @@ build:cuda_clang --copt=-Wno-gnu-offsetof-extensions
 # Disable clang extention that rejects unknown arguments.
 build:cuda_clang --copt=-Qunused-arguments
 
+build:mosaic_gpu --@llvm-project//mlir:enable_cuda=true
```

Comment:
The target `enable_cuda` seems not existed when I tried to build mosaic_gpu from the latest commit.
```
>> python build/build.py --enable_cuda --enable_mosaic_gpu
...
no such target '@llvm-project//mlir:enable_cuda': target 'enable_cuda' not declared in package 'mlir'
...
```

Reply:
Figured it out: it needs a pending llvm PR: https://github.com/llvm/llvm-project/pull/88856